packages/viz/src/utils/data.js

```diff
@@ -109,7 +108,17 @@ class DataUtil {
 
       this.filter.byActiveDays(activeDays);
 
-      this.days = daysInRange / 7 * activeDays.length;
+      console.log(activeDays);
+
+      // here is a more realistic version of calculating days, fixing a bug in stats
+      this.days = 0;
+      let start = moment(this._endpoints[0]);
```

Comment:
```suggestion
      let start = moment.utc(this._endpoints[0]);
```
Or use the timezone. But never use moment() directly